postgres-exporter/main.libsonnet

```diff
@@ -0,0 +1,43 @@
+local k = import 'ksonnet-util/kausal.libsonnet';
+
+{
+  new(
+    name,
+    data_source_name='$(USER):$(PASSWORD)@tcp($(HOST):$(PORT))/',
```

Comment:
First time seeing `@tcp($(HOST):$(PORT))/` - and not finding any references in https://github.com/prometheus-community/postgres_exporter/ or https://github.com/lib/pq.

Curious: how does this differ from `@$(HOST):$(PORT)/` ?

Reply:
Good catch, I based this lib on mysqld_exporter, it parses `unix()` and `tcp()` bits in the data_source_name, I assumed this was standard net/url parser behavior but apparently it is not. Lemme fix.

Reply:
```suggestion
    data_source_name='$(USER):$(PASSWORD)@$(HOST):$(PORT)/',
```